cmd/lakectl/cmd/root.go

```diff
@@ -216,25 +216,50 @@ func withSyncFlags(cmd *cobra.Command) {
 	withNoProgress(cmd)
 }
 
+func getStorageConfigOrDie(ctx context.Context, client *apigen.ClientWithResponses, repositoryID string) *apigen.StorageConfig {
+	confResp, err := client.GetConfigWithResponse(ctx)
+	DieOnErrorOrUnexpectedStatusCode(confResp, err, http.StatusOK)
+
+	storageConfigList := confResp.JSON200.StorageConfigList
+	if storageConfigList != nil && len(*storageConfigList) > 1 {
```

Comment:
And if it's one? (Sorry for missing it last time)

Reply:
If it's one, `StorageConfig` will also be populated, so better using it.

Reply:
It's not consistant and confusing, if we're using blockstores please use blockstores